File: Javascript/1.twoSum_I.js
/*
 * @lc app=leetcode.cn id=1 lang=javascript
 *
 * [1] 两数之和
 */

// @lc code=start
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {
  var lens = nums.length;
  for (var i = 0; i < lens; i++) {
    for (var j = i + 1; j < lens; j++) {
      if (nums[i] + nums[j] == target) {
        return [i, j];
      }
    }
  }

  return [];
};
// @lc code=end

File: Golang/1.twoSum_I.go
package twosum

/*
 * @lc app=leetcode.cn id=1 lang=golang
 *
 * [1] 两数之和
 */

// @lc code=start

func twoSum(nums []int, target int) []int {

	var result []int

	numsLength := len(nums)

	for i := 0; i < numsLength; i++ {
		for j := i + 1; j < numsLength; j++ {
			if nums[i]+nums[j] == target {
				result[0] = i
				result[1] = j

				return result
			}
		}
	}

	return nil
}

// @lc code=end
